run.cmd

```diff
@@ -23,6 +23,9 @@ set _json=%~dp0config.json
 
 pushd "%~dp0"
 call "%_dotnet%" "%_toolRuntime%\run.exe" "%_json%" %*
+
+:: Terminate all dotnet build processes.
+call "%_dotnet%" "build-server" "shutdown"
```

Comment:
This is not where I would expect this line to go.

For things like this (that only affect official build, or CI, etc), we should be modifying the official build and/or CI .yml files. This kind of change affects EVERYONE who tries building our repo - even devs on their own boxes.  Killing the build server after each time I call "build" on the command line slows down my dev workflow because it needs to spin the build server up again the next time I call "build".

Reply:
Ok I will change that to the .yml file so that it only affects the official build.